pages/osx/ping.md

```diff
@@ -0,0 +1,27 @@
+# ping
+
+> Send ICMP ECHO_REQUEST packets to network hosts.
+
+- Ping `host`:
```

Comment:
The backticks aren't necessary here afaik, it's only really for command syntax highlighting.

Perhaps the following would sound better?
```suggestion
- Ping the specified host:
```